client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -5,83 +5,119 @@ import Filters from '../../components/Filter/Filter';
+    return `https://c.tile.openstreetmap.org/${z}/${x}/${y}.png`;
+  }
+
+  const markers = [
+    [50.874, 4.6947],
+    [50.212, 4],
+    [51, 5],
+  ];
+
+  function markerClick({ anchor }) {
+    console.log(anchor);
+  }
 
   function renderView() {
-    if (listOrMap == true) {
+    if (listMode == true) {
       return (
         <div>
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
+          <div className="list-mode">
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+          </div>
```

Comment:
sorry i probably should've mentioned this last PR, but instead of one big function `renderView` can you put this in a function like `renderListView`?